app/config/sql/user/all.sql

```diff
@@ -0,0 +1 @@
+//This file and folder will hold all sql commands to be performed on the database
```

Comment:
Hey, sql files can be commented out using /* for multiple lines or for a single line you can use --